src/main/java/com/phonebookservice/controller/ContactController.java

```diff
@@ -0,0 +1,55 @@
+package main.java.com.phonebookservice.controller;
+
+import main.java.com.phonebookservice.exception.PhoneBookControllerException;
+import main.java.com.phonebookservice.model.Contact;
+import main.java.com.phonebookservice.model.Model;
+import main.java.com.phonebookservice.server.DBIf;
+import main.java.com.phonebookservice.util.Messages;
+import main.java.com.phonebookservice.util.Util;
+
+public class ContactController extends AbstractController {
+
+    /**
+     * contact controller constructor.
+     *
+     * @param databse the database
+     */
+    public ContactController(final DBIf databse) {
+        super(databse);
+    }
+
+    @Override
+    public final void create(final Model model) {
+        final DBIf database = getDatabase();
```

Comment:
if you make it protected, you don't need to call `getDatabase()` method

Reply:
we decided to leave the method:  getDatabase but protected.

Reply:
yes +1